index.html

```diff
@@ -1,59 +1,133 @@
-      align-content:center;
-      justify-content:center;
-    flex-direction:row;
-    margin: 0 !important;
-      flex-basis:auto;
-	   }
-	   img {
-		  width: 100%;
-		  margin: 20px;
-	   }
-	   h1 {
-	       font-size: 3em;
-	       color: white;
-	       padding-top:80px;
-	       /*background-color: rgba(183, 58, 98, 0.89);*/
-	   }
-	</style>
-</head>
-
-<body>
-	<!--<img style ="width: 80px; height: 80px; margin-top:10px" src ="./images/logo.png">-->
-    <h1 >The Coding Space Projects</h1>
-   <div id="container">
-	<div>
-		<a href="./scratch"><img style ="width:300px; height: 300px" id="banner1" src="./images/scratch2.png" onmouseover="this.src='./images/scratchhover.png'" onmouseout="this.src='./images/scratch2.png'" /></a>
-	</div>
-	<div>
-		<a href="http://woofjs.com/create"><img  style ="width:300px; height: 300px" id="banner3" src="./images/woof2.png" onmouseover="this.src='./images/woofhover.png'" onmouseout="this.src='./images/woof2.png'"/></a>
-	</div>
-	<div>
-	<a href="./web"><img  style ="width:300px; height: 300px" id="banner2" src="./images/web2.png" onmouseover="this.src='./images/webhover.png'" onmouseout="this.src='./images/web2.png'"/></a>
-	</div>
-</div>
-</body>
-
-</html>
-</div>
+
+	<head>
+		<meta charset="utf-8">
+		<meta name="description" content="">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<title>The Coding Space Projects</title>
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+		<link rel="stylesheet" href="index.css">
+		<link rel="icon" type="favicon/x-icon" href="images/favicon.ico">
+		<script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"></script>
+   </head>
+
+   <body>
+
+   	<h1>The Coding Space Projects</h1>
+   	<div class="container">
+
+	   	<div class="row tcs-row">
+	   		<div class="col-sm-1"></div>
+		   		<div class="col-sm-10 holder">
+		   			<a id="scratch-img" href="./scratch"><img class="img-responsive" src="./images/scratch2.png" alt="Scratch" /></a>
+		   			<a id="woof-img" href="http://woofjs.com/create"><img class="img-responsive" src="./images/woof2.png" alt="Woof" /></a>
+		   			<a id="web-img" href="./web"><img class="img-responsive" src="./images/web2.png" alt="Scratch" /></a>
+		   		</div>
+		   	<div class="col-sm-1"></div>
+	   	</div>
+
+	   	<div class="row tcs-row hidden-xs">
+	   		<div class="col-sm-1"></div>
+		   		<div class="col-sm-10" id="platform-holder">
+		   			<div class="textbox-holder">
+		   				<div id="scratch-box" class="black-opaque infobox">Program your own interactive stories, games, and animations with Scratch -- and share your creations with others in the online community. <p class="info-para">We recommend starting here if you're new to coding.</p>  </div>
```

Comment:
Maybe put the "start here" line in green? And the respective line about pre-reqs on Woof in yellow, and for Web in red?